<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> 
<html> <!--<![endif]-->
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  <meta name="description" content="RECOMB 2025 - Yonsei University, Seoul">
  
  <meta name="author" content="Themefisher.com">

  <title>RECOMB 2025 | PROGRAM</title>

  <link rel="alternate" href="http://recomb2024.bilkent.edu.tr" hreflang="en-gb" />
  <link rel="alternate" href="http://recomb2024.bilkent.edu.tr" hreflang="en-us" />

<!-- Mobile Specific Meta
  ================================================== -->
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  <!-- Favicon -->
  <link rel="shortcut icon" type="image/x-icon" href="images/favicon.png" />
  
  <!-- CSS
  ================================================== -->
  <!-- Themefisher Icon font -->
  <link rel="stylesheet" href="plugins/themefisher-font/style.css">
  <!-- bootstrap.min css -->
  <link rel="stylesheet" href="plugins/bootstrap/css/bootstrap.min.css">
  <!-- Lightbox.min css -->
  <link rel="stylesheet" href="plugins/lightbox2/dist/css/lightbox.min.css">
  <!-- animation css -->
  <link rel="stylesheet" href="plugins/animate/animate.css">
  <!-- Slick Carousel -->
  <link rel="stylesheet" href="plugins/slick/slick.css">
  <!-- Main Stylesheet -->
  <link rel="stylesheet" href="css/style.css">  
  
  <script src="https://kit.fontawesome.com/5ce389c680.js" crossorigin="anonymous"></script>

</head>

<body id="body">

 <!-- Start Preloader
  ==================================== -->
  <div id="preloader">
    <div class='preloader'>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
      <span></span>
    </div>
  </div> 
<!-- End Preloader
  ==================================== -->

<!-- Fixed Navigation
  ==================================== -->
  <header class="navigation fixed-top">
	<div class="container">
	  <!-- main nav -->
	  
	  <nav class="navbar navbar-expand-lg navbar-light">
		<!-- logo -->
		<a class="navbar-brand logo" href="index.html">
		  <img class="logo-default" src="images/recomb-logo/logo-name.svg" style="height:60px;" alt="RECOMB Conference Logo"/>
		  <img class="logo-white" src="images/recomb-logo/logo-name-white.svg" style="height:60px;" alt="RECOMB Conference Logo"/>
		</a>
		<!-- /logo -->
		<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navigation"
		  aria-controls="navigation" aria-expanded="false" aria-label="Toggle navigation">
		  <span class="navbar-toggler-icon"></span>
		</button>
  
		<div class="collapse navbar-collapse" id="navigation">
		  <ul class="navbar-nav mr-auto text-center">
			<li class="nav-item ">
			  <a class="nav-link" href="index.html">Home</a>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Conference</a>
			  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="committees.html">Committees</a>
				<a class="dropdown-item" href="keydates_and_deadlines.html">Key dates & Deadlines</a>
				<a class="dropdown-item" href="call_for_papers.html">Call for Papers</a>
				<a class="dropdown-item" href="call_for_highlights.html">Call for Highlights</a>
              <a class="dropdown-item" href="call_for_posters.html">Call for Posters</a>			  </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Register</a>
			  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="https://www.recomb2025.com/html/">RECOMB and Satellites</a>
				<a class="dropdown-item" href="travel_fellowships.html">Travel Fellowships</a>
			  </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Program</a>
			  <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="program.html">Program Schedule</a>
				<a class="dropdown-item" href="keynotes.html">Keynotes</a>
				<a class="dropdown-item" href="posters.html">Poster Guidelines</a>
			  </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Venue</a>
			   <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="venue_main.html">Main Conference Venue</a>
				<a class="dropdown-item" href="venue_satellites.html">Satellites Venue</a>
				<a class="dropdown-item" href="hotels.html">Hotels</a>
				<a class="dropdown-item" href="airport.html">Airport Directions</a>
			<a class="dropdown-item" href="things_to_do.html">Things to do in Seoul</a>
				<a class="dropdown-item" href="transportation.html">Local Transportation</a>
			<a class="dropdown-item" href="visa.html">Visa Information</a>
			  </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Satellites</a>
			   <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				   <a class="dropdown-item" href="satellites_schedule.html">Satellites Schedule</a>
		              <a class="dropdown-item" href="https://recomb-seq.github.io/" target="_blank">RECOMB–Seq</a>
		              <a class="dropdown-item" href="https://recomb-cg-2025.github.io/" target="_blank">RECOMB–CG</a>
		              <a class="dropdown-item" href="recombgenetics.html">RECOMB–Genetics</a>
		              <a class="dropdown-item" href="recombccb.html">RECOMB–CCB</a>
		              <a class="dropdown-item" href="recombrsg.html">RECOMB–CSB</a>
		              <a class="dropdown-item" href="recombmicrobiome.html">RECOMB–Microbiome</a>
		              <a class="dropdown-item" href="https://recomb-privacy-equity.github.io/">RECOMB–P&E</a>            </div>
			</li>
			<li class="nav-item dropdown active">
			  <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown"
				aria-haspopup="true" aria-expanded="false">Sponsors</a>
			   <div class="dropdown-menu" aria-labelledby="navbarDropdown">
				<a class="dropdown-item" href="sponsors.html">RECOMB Sponsors</a>
				<a class="dropdown-item" href="sponsorship_packages.html">Sponsorship Packages</a>
			  </div>
			</li>
			<li class="nav-item ">
			  <a href="https://www.recomb.org" target="blank" style="color:white;">About RECOMB</a>
			</li>
		  </ul>
		</div>
	  </nav>
	  <!-- /main nav -->
	</div>
  </header>
  <!-- End Fixed Navigation
	==================================== -->
<section class="single-page-header">
	<div class="container">
		<div class="row">
			<div class="col-md-12">
				<h2>Program</h2>
			</div>
		</div>
	</div>
</section>

<!-- Start Program
    ==================================== -->
<section class="team-skills section-sm" id="skills">
  <div class="container">
    <div class="col-12">
      <div class="team-skills-content center">
        <hr> <h2>Program Schedule</h2> <hr>
		<b><a href="accepted_papers.html">RECOMB 2025 Accepted Papers</a></b>
	<!--	<b> [PT] for proceedings talk, [HT] for highlights talk.</b> The agenda is also available on our Whova installation.<br><br><br>
		<a href="https://link.springer.com/book/10.1007/978-3-031-29119-7">RECOMB 2025 Conference Proceedings</a>. Link will be active for conference attendees after April 10, 2024 until May 10, 2024.
		<br>		<br>
		<center>
		
		<a href="https://whova.com/portal/aicrc_202404/?source=btn_link" title="Event management software" id="whova-buttonlink" target="_blank" style="background-color:#2DACEE; padding:10px; border-radius:5px; color:#fff; font-size:16px"/>Get Whova Now</a>
		</center>
      </div>
	  </div>
	-->
<section class="program section" id="program">
  <div class="container">
    <div class="row justify-content-center">
      <div class="col-12">
        <h3>Day 1 (April 26, 2025)</h3>
          <hr>
	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">08:45 - 09:05</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity"><b>Opening Ceremony</b></p></div>
          </div>

          <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:05 - 10:05</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker"><a href="https://people.seas.harvard.edu/~valiant/">Leslie Valiant</a></p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
		<button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote1"> Keynote Talk</button>
          </div>
          </div>
          
    <div class="row">
    <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
    <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote1" class="collapse show">

      <p class="text">
          <b>Darwinian Evolution as a Form of Learning</b>
          <br><br>
          <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek1"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
          <div class="collapse" id="collapsek1">
            <p class="text">
              Living organisms function according to protein circuits. Darwin's theory of evolution suggests that these circuits have evolved through variation guided by natural selection. However, it is currently not understood how variation mechanisms can give rise, within realistic population sizes and realistic numbers of generations, to protein circuits of the complexity found in nature. 

              We suggest that machine learning offers the framework for investigating this question of how complex circuits can come into being via a Darwinian process without a designer. We formulate evolution as a form of learning from examples. The targets of the learning process are the protein expression functions that come close to best behavior in the specific environment. The learning process is constrained so that the feedback from experience is Darwinian. We formulate a notion of evolvability that distinguishes function classes that are evolvable with polynomially bounded resources from those that are not. The dilemma is that if the function class that describes the expression levels of proteins in terms of each other, is too restrictive, then it will not support biology, while if it is too expressive then no evolution algorithm will exist to navigate it. 
            </p>
          </div>
      </p>
      </div></div>
    </div>  

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:05 - 10:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Coffee Break</p></div>
          </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:20 - 11:14</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session1">Session 1: Protein Function</button>
		<div id="session1" class="collapse show">
		<p class="text">
                <b>[PT] An adversarial scheme for integrating multi-modal data on protein function</b>
                <br>
                <i>Rami Nasser, Leah Schaffer, Trey Ideker and Roded Sharan</i>
                <hr>

		<p class="text">
                <b>[PT] Learning maximally spanning representations improves protein function annotation</b>
                <br>
                <i>Jiaqi Luo and Yunan Luo</i>
                <hr>

		<p class="text">
                <b>[PT] DualGOFiller: A dual-channel graph neural network with contrastive learning for enhancing function prediction in partially annotated proteins</b>
                <br>
                <i>Shaojun Wang, Hancheng Liu, Weiqi Zhai and Shanfeng Zhu</i>
			
          	</div>
          </div>
	  </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">11:15 - 12:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session2">Session 2: Networks and Interactions</button>
		<div id="session2" class="collapse show">
		<p class="text">
                <b>[PT] Decoding the Functional Interactome of Non-Model Organisms with PHILHARMONIC</b>
                <br>
                <i>Samuel Sledzieski, Charlotte Versavel, Rohit Singh, Faith Ocitti, Kapil Devkota, Lokender Kumar, Polina Shpilker, Liza Roger, Jinkyu Yang, Nastassja Lewinski, Hollie Putnam, Bonnie Berger, Judith Klein-Seetharaman and Lenore Cowen</i>
                <hr>

		<p class="text">
                <b>[PT] STEAMBOAT: Attention-based multiscale delineation of cellular interactions in tissues</b>
                <br>
                <i>Shaoheng Liang, Junjie Tang, Guanghan Wang and Jian Ma</i>
                <hr>

		<p class="text">
                <b>[HT] Inferring gene regulatory networks from single-cell multiome data using atlas-scale external data</b>
                <br>
                <i>Zhana Duren and Qiuyue Yuan</i>
                <hr>

		<p class="text">
                <b>[HT] DD-PRiSM: a deep learning framework for decomposition and prediction of synergistic drug combinations</b>
                <br>
                <i>Iljung Jin, Songyeon Lee, Martin Schmuhalek and Hojung Nam</i>
			
          	</div>
          </div>
	  </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">12:20 - 13:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Lunch Break</p></div>
          </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">13:30 - 14:30</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker"><a href="https://biosci.snu.ac.kr/en/people/faculty?mode=view&profidx=5">V. Narry Kim</a></p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
	  	<button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote2"> Keynote Talk</button>
          </div>
          </div>
    <div class="row">
    <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
    <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote2" class="collapse show">

      <p class="text">
          <b>How to deal with exogenous RNAs: From viral RNAs to mRNA therapeutics</b>
          <br><br>
          <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek2"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
          <div class="collapse" id="collapsek2">
            <p class="text">
              RNAs of external origin, such as viral RNAs and therapeutic mRNAs, rely on cellular machinery for entry and translation while facing cellular barriers that restrict their functions. Thus, for developing effective antivirals and RNA therapeutics, it is important to understand how cells deal with RNAs. In this presentation, I will discuss two recent studies exploring the regulatory mechanisms of exogenous RNAs. 

              In the first part, I will talk about our studies on viral RNA regulation. Using massively parallel reporter assays of viral genomic segments, we discovered hundreds of RNA elements that control RNA stability and translation. Investigation of their mechanisms provides new insights into the regulation of both viral and cellular RNAs. This research creates a valuable resource while highlighting the potential of viral genomes for biological discovery and therapeutic development. 
              
              Second, I will present our recent work on mRNA vaccines. Through genome-wide screens of in vitro-transcribed (IVT) mRNAs encapsulated in lipid nanoparticles (LNPs), we identified key cellular factors that impact mRNA. By comparing mRNAs with and without N1-methylpseudouridine modification, we also uncovered the mechanism by which this modification enhances protein production from IVT mRNAs. Our study provides a comprehensive map of cellular pathways regulating exogenous mRNAs, offering insights for improving RNA therapeutics.
              
            </p>
          </div>
      </p>
      </div></div>
    </div>  


	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">14:30 - 15:25</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session3">Session 3: Evolutionary and Lineage Inference</button>
		<div id="session3" class="collapse show">
		<p class="text">
                <b>[PT] An Exact and Fast SAT Formulation for the DCJ Distance</b>
                <br>
                <i>Aaryan Mahesh Sarnaik, Ke Chen, Austin Diaz and Mingfu Shao</i>
                <hr>

		<p class="text">
                <b>[PT] A k-mer-based maximum likelihood method for estimating distances of reads to genomes enables genome-wide phylogenetic placement</b>
                <br>
                <i>Ali Osman Berk Şapcı and Siavash Mirarab</i>
                <hr>

		<p class="text">
                <b>[PT] Characterizing the solution space of migration histories of metastatic cancers with MACH2</b>
                <br>
                <i>Mrinmoy Saha Roddur, Vikram Ramavarapu, Abi Bunkum, Ariana Huebner, Roman Mineyev, Nicholas McGranahan, Simone Zaccaria and Mohammed El-Kebir</i>
        
          	</div>
	  </div>
	  </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:25 - 15:40</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Coffee Break</p></div>
          </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:40 - 16:31</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session4">Session 4: RNA Structure</button>
		<div id="session4" class="collapse show">
		<p class="text">
                <b>[PT] Old dog, new tricks: Exact seeding strategy improves RNA design performances</b>
                <br>
                <i>Théo Boury, Leonhard Sidl, Ivo L. Hofacker, Yann Ponty and Hua-Ting Yao</i>
                <hr>

		<p class="text">
                <b>[PT] Scalable and interpretable identification of minimal undesignable RNA structure motifs with rotational invariance</b>
                <br>
                <i>Tianshuo Zhou, Wei Yu Tang, Apoorv Malik, David H. Mathews and Liang Huang</i>
                <hr>

		<p class="text">
                <b>[HT] Accurate RNA 3D structure prediction using a language model-based deep learning approach</b>
                <br>
                <i>"Tao Shen, Zhihang Hu, Siqi Sun, Di Liu, Felix Wong, Jiuming Wang, Jiayang Chen, Yixuan Wang, Liang Hong, Jin Xiao, Liangzhen Zheng, Tejas Krishnamoorthi, Irwin King, Sheng Wang, Peng Yin, James J. Collins and Yu Li"</i>
        
          	</div>
	  </div>
	  </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">16:31 - 17:43</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session5">Session 5: AI and Biomedicine</button>
		<div id="session5" class="collapse show">
		<p class="text">
                <b>[PT] Hierarchical spatio-temporal state-space modeling for fMRI analysis</b>
                <br>
                <i>Yuxiang Wei, Anees Abrol and Vince Calhoun</i>
                <hr>

		<p class="text">
                <b>[PT] Sequence-based TCR-peptide representations using cross-epitope contrastive fine-tuning of protein language models</b>
                <br>
                <i>Chiho Im, Ryan Zhao, Scott Boyd and Anshul Kundaje</i>
                <hr>

		<p class="text">
                <b>[PT] A phylogenetic approach to genomic language modeling</b>
                <br>
                <i>Carlos Albors, Jianan Canal Li, Gonzalo Benegas, Chengzhong Ye and Yun S. Song</i>
                <hr>

		<p class="text">
                <b>[PT] Rag2Mol: Structure-based drug design based on Retrieval Augmented Generation</b>
                <br>
                <i>Peidong Zhang, Xingang Peng, Rong Han, Ting Chen and Jianzhu Ma</i>
        
          	</div>
	  </div>
	  </div>
	      
	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">17:45 - 19:15</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Poster Session I and Coffee Break</p></div>
          </div>

        <hr>
	<h3>Day 2 (April 27, 2025)</h3>
        <hr>   

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:00 - 09:05</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity"><b>Welcome</b></p></div>
          </div>

          <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:05 - 10:05</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker"><a href="https://www.embl.org/people/person/bork/">Peer Bork</a></p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
		    <button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote3"> Keynote Talk</button>
          </div>
          </div>
          <div class="row">
            <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
            <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote3" class="collapse show">
        
              <p class="text">
                  <b>Microbiome analysis for human and planetary health</b>
                  <br><br>
                  <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek3"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                  <div class="collapse" id="collapsek3">
                    <p class="text">
                      Based on computational methods and resources, often developed in our group, here (i) I first introduce a concept of federating such interoperable bioinformatics tools and resources for efficient and flexible large scale data analysis. Applied to environmental sequencing, that is metagenomics, which has become a major driver for uncovering microbial biodiversity and increasingly also for molecular functionality on our planet, it enables powerful microbiome analysis. (ii) I illustrate this by our work on the gut microbiome, arguable the best-studied microbial community, serving as a model for other habitats. Metagenome-wide association studies enable bioinformatics-driven modelling of the microbial communities, but modelling needs constant extension and refinement, e.g. inclusion of eukaryotes or the incorporation of “absolute” abundance. (iii) I further show how to apply the underlying analysis concepts to other habitats, like ocean and soil, to arrive at a basic understanding of microbial life on earth, e.g. of gene evolution at global scale or of fluxes of molecular functions across habitats. For validation, but also hypothesis generation, we complement public data with newly generated ones from a large continental-scale, international expedition that traversed European coastlines (TREC). To maximize the added value of the new data, they need to be integrated with historic ones of very different data types. For this we are developing and utilizing SPIRE, a searchable, planetary-scale microbiome resource, for a baseline understanding the global microbiome structure and function as well as for microbiome-informed bioindicator and bioremediation applications towards improving planetary health. 
                    </p>
                  </div>
              </p>
              </div></div>
            </div>  

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:05 - 10:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Coffee Break</p></div>
          </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:20 - 12:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session6">Session 6: Sequence Analysis</button>
		<div id="session6" class="collapse show">
		<p class="text">
                <b>[PT] devider: long-read reconstruction of many diverse haplotypes</b>
                <br>
                <i>Jim Shaw, Christina Boucher, Yun William Yu, Noelle Noyes and Heng Li</i>
                <hr>

		<p class="text">
                <b>[PT] Hyper-k-mers: efficient streaming k-mers representation</b>
                <br>
                <i>Igor Martayan, Lucas Robidou, Yoshihiro Shibuya and Antoine Limasset</i>
                <hr>

		<p class="text">
                <b>[PT] Improved pangenomic classification accuracy with chain statistics</b>
                <br>
                <i>Nathaniel K. Brown, Vikram Shivakumar and Ben Langmead</i>
                <hr>

		<p class="text">
                <b>[PT] Integer programming framework for pangenome-based genome inference</b>
                <br>
                <i>Ghanshyam Chandra, Md Helal Hossen, Stephan Scholz, Alexander T Dilthey, Daniel Gibney and Chirag Jain</i>
                <hr>

		<p class="text">
                <b>[PT] Prokrustean Graph: A substring index for rapid k-mer size analysis</b>
                <br>
                <i>Adam Park and David Koslicki</i>
                <hr>

		<p class="text">
                <b>[HT] SigAlign: an alignment algorithm guided by explicit similarity criteria</b>
                <br>
                <i>Kunhyung Bahk and Joohon Sung</i>
                <hr>

		<p class="text">
                <b>[HT] Fast, sensitive detetion of protein homologs using deep dense retrieval</b>
                <br>
                <i>Liang Hong, Yu Li, Zhihang Hu and Jiuming Wang</i>
        
          	</div>	
	  </div>
	  </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">12:20 - 13:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Lunch Break</p></div>
          </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">13:30 - 14:30</p></div>
	  <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker"><a href="https://www.cs.princeton.edu/people/profile/ogt">Olga Troyanskaya</a></p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
		    <button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote4"> Keynote Talk</button>
          </div>
          </div>

          <div class="row">
            <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
            <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote4" class="collapse show">
        
              <p class="text">
                  <b>TBD</b>
                  <br><br>
                  <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek4"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                  <div class="collapse" id="collapsek4">
                    <p class="text">
          TBD
                    </p>
                  </div>
              </p>
              </div></div>
            </div>  


	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">14:30 - 15:24</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session7">Session 7: Genetic Variation</button>
		<div id="session7" class="collapse show">
		<p class="text">
                <b>[PT] ScatTR: Estimating the size of long tandem repeat expansions from short-reads</b>
                <br>
                <i>Rashid Al-Abri and Gamze Gürsoy</i>
                <hr>

		<p class="text">
                <b>[PT] OMKar: optical map based automated karyotyping of genomes to identify constitutional abnormalities</b>
                <br>
                <i>Siavash Raeisi Dehkordi, Zhaoyang Jia, Joey Estabrook, Jen Hauenstein, Neil Miller, Paul Dremsek, Alex Hastie, Andy Wing Chun Pang and Vineet Bafna</i>
                <hr>

		<p class="text">
                <b>[PT] Accurate detection of tandem repeats from error-prone long reads with EquiRep</b>
                <br>
                <i>Zhezheng Song, Tasfia Zahin, Xiang Li and Mingfu Shao</i>
        
          	</div>		
	  </div>
	  </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:25 - 15:40</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Coffee Break</p></div>
          </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:40 - 17:04</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session8">Session 8: Precision Medicine</button>
		<div id="session8" class="collapse show">
		<p class="text">
                <b>[PT] cfDecon: Accurate and Interpretable methylation-based cell type deconvolution for cell-free DNA</b>
                <br>
                <i>Yixuan Wang, Jiayi Li, Jingqi Li, Shen Yang, Yuhan Huang, Xinyuan Liu, Yimin Fan, Irwin King, Yumei Li and Yu Li</i>
                <hr>

		<p class="text">
                <b>[PT] ALPINE: an interpretable approach for decoding phenotypes from multicondition sequencing data</b>
                <br>
                <i>Wei-Hao Lee, Lechuan Li, Ruth Dannenfelser and Vicky Yao</i>
                <hr>
			
		<p class="text">
                <b>[PT] Learning multi-cellular representations of single-cell transcriptomics data enables characterization of patient-level disease states</b>
                <br>
                <i>Tianyu Liu, Edward De Brouwer, Tony Kuo, Nathaniel Diamant, Alsu Missarova, Hanchen Wang, Minsheng Hao, Hector Corrada Bravo, Gabriele Scalia, Aviv Regev and Graham Heimberg</i>
		<hr>
			
		<p class="text">
                <b>[HT] Coordinated, multicellular patterns of transcriptional variation that stratify patient cohorts are revealed by tensor decomposition</b>
                <br>
                <i>Jonathan Mitchel and Peter Kharchenko</i>
		<hr>
						
		<p class="text">
                <b>[HT] Machine learning-guided co-optimization of fitness and diversity facilitates combinatorial library design in enzyme engineering</b>
                <br>
                <i>Kerr Ding, Michael Chin, Yunlong Zhao, Wei Huang, Binh Khanh Mai, Huanan Wang, Peng Liu, Yang Yang and Yunan Luo</i>
        
          	</div>		  
	  </div>
	  </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">17:05 - 17:56</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session9">Session 9: Phylogenetics</button>
		<div id="session9" class="collapse show">
		<p class="text">
                <b>[PT] Untying rates of gene gain and loss leads to a new phylogenetic approach</b>
                <br>
                <i>Yoav Dvir and Sagi Snir</i>
                <hr>

		<p class="text">
                <b>[PT] The tree labeling polytope: a unified approach to ancestral reconstruction problems</b>
                <br>
                <i>Henri Schmidt and Benjamin Raphael</i>
                <hr>

		<p class="text">
                <b>[HT] CASTER: Direct species tree inference from whole-genome alignments</b>
                <br>
                <i>Chao Zhang, Rasmus Nielsen and Siavash Mirarab</i>
        
          	</div>
	  </div>
	  </div>
	      
	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">18:00 - 19:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Poster Session II and Coffee Break</p></div>
          </div>
	      
        <hr>
        <h3>Day 3 (April 28, 2025)</h3>
        <hr>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:00 - 09:05</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity"><b>Welcome</b></p></div>
          </div>

          <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:05 - 10:05</p></div>
	  <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker"><a href="https://www.bbe.caltech.edu/people/lior-s-pachter">Lior Pachter</a></p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
		    <button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote5"> Keynote Talk</button>
          </div>
          </div>

          <div class="row">
            <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
            <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote5" class="collapse show">
        
              <p class="text">
                  <b>A controlled approach to computational biology via axiomatic derivations</b>
                  <br><br>
                  <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek5"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                  <div class="collapse" id="collapsek5">
                    <p class="text">
                      Methods in computational biology have varied origins. Some have been derived as solutions to desired optimizations, while others have emerged as successful heuristics, whose theoretical underpinnings were understood only after widespread adoption. I will review several examples ranging from methods in phylogenetics to tools for studying gene regulation, and will argue for an axiomatic approach, whose benefits mimic those of conducting comprehensive controls in experimental biology.
                    </p>
                  </div>
              </p>
              </div></div>
            </div>  

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:05 - 10:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Coffee Break</p></div>
          </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:20 - 11:32</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session10">Session 10: Statistical Approaches</button>
		<div id="session10" class="collapse show">
		<p class="text">
                <b>[PT] Antimicrobial drug recommendation from MALDI-TOF mass spectrometry with statistical guarantees using conformal prediction</b>
                <br>
                <i>Nina Corvelo Benz, Lucas Miranda, Dexiong Chen, Janko Sattler and Karsten Borgwardt</i>
                <hr>

		<p class="text">
                <b>[PT] mcRigor: a statistical method to enhance the rigor of metacell partitioning in single-cell data analysis</b>
                <br>
                <i>Pan Liu and Jingyi Jessica Li</i>
                <hr>

		<p class="text">
                <b>[PT] ML-MAGES: A machine learning framework for multivariate genetic association analyses with genes and effect size shrinkage</b>
                <br>
                <i>Xiran Liu, Lorin Crawford and Sohini Ramachandran</i>
                <hr>
			
		<p class="text">
                <b>[PT] Learning Latent Trajectories in Developmental Time Series with Hidden-Markov Optimal Transport</b>
                <br>
                <i>Peter Halmos, Julian Gold, Xinhao Liu and Benjamin Raphael</i>
        
          	</div>		  
	  </div>
	  </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">11:32 - 12:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#sessionSP">Special Session: Korean Big Data Bioscience</button>        
	  </div>
	  </div>	      

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">12:20 - 13:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Lunch Break</p></div>
          </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">13:30 - 14:30</p></div>
	  <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker"><a href="https://kaist.ac.kr/leaderen/html/vicepresident/0202.html">Sang Yup Lee</a></p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
		    <button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote6"> Keynote Talk</button>
          </div>
          </div>
          <div class="row">
            <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
            <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote6" class="collapse show">
        
              <p class="text">
                  <b>Systems Metabolic Engineering of Bacteria for Natural Products</b>
                  <br><br>
                  <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek6"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                  <div class="collapse" id="collapsek6">
                    <p class="text">
                      Systems metabolic engineering is a comprehensive field that combines the principles of traditional metabolic engineering with the cutting-edge techniques of systems biology, synthetic biology, and evolutionary engineering to enhance the development of bacterial cell factories. In today's lecture, I will detail the various strategies used in systems metabolic engineering to develop these bacterial cell factories. Additionally, I will discuss the specific systems metabolic engineering strategies that have been utilized for the development of strains aimed at producing a range of products, with a particular focus on natural products. These strategies not only streamline the production processes but also improve the efficiency and yield of the desired products.
                    </p>
                  </div>
              </p>
              </div></div>
            </div>  

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">14:30 - 15:25</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session11">Session 11: Single-cell (I)</button>
		<div id="session11" class="collapse show">
		<p class="text">
                <b>[PT] TarDis: Achieving robust and structured disentanglement of multiple covariates</b>
                <br>
                <i>Kemal Inecik, Aleyna Kara, Antony Rose, Muzlifah Haniffa and Fabian Theis</i>
                <hr>

		<p class="text">
                <b>[PT] Optimal marker genes for c-separated cell types</b>
                <br>
                <i>Bartol Borozn, Luka Borozan, Domagoj Severdija, Domagoj Matijevic and Stefan Canzar</i>
                <hr>

		<p class="text">
                <b>[PT] Causal disentanglement of treatment effects in single-cell RNA Sequencing through counterfactual inference </b>
                <br>
                <i>Shaokun An, Jae-Won Cho, Kai Cao, Jiankang Xiong, Martin Hemberg and Lin Wan</i>
        
          	</div>		  
	  </div>
	  </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:25 - 15:40</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Coffee Break</p></div>
          </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:40 - 16:52</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session12">Session 12: Protein and Small Molecules</button>
		<div id="session12" class="collapse show">
		<p class="text">
                <b>[PT] Orientation-aware graph neural networks for protein structure representation learning</b>
                <br>
                <i>Jiahan Li, Shitong Luo, Cengyue Deng, Chaoran Cheng, Jiaqi Guan, Leonidas Guibas, Jian Peng and Jianzhu Ma</i>
                <hr>

		<p class="text">
                <b>[PT] Active learning for protein structure prediction</b>
                <br>
                <i>Zexin Xue, Michael Bailey, Abhinav Gupta, Alejandro Corrochano-Navarro, Sizhen Li, Ruijiang Li, Qiu Yu, Ziv Bar-Joseph, Sven Jager and Lorenzo Kogler-Anele</i>
                <hr>

		<p class="text">
                <b>[PT] Rewiring protein sequence and structure generative models to enhance protein stability prediction</b>
                <br>
                <i>Ziang Li and Yunan Luo</i>
                <hr>
			
		<p class="text">
                <b>[PT] Learning a CoNCISE language for small molecule binding and function</b>
                <br>
                <i>Mert Erden, Kapil Devkota, Lia Varghese, Lenore Cowen and Rohit Singh</i>
        
          	</div>		  
	  </div>
	  </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">18:15 - 19:00</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Business Meeting</p></div>
          </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">19:00</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Gala Dinner</p></div>
          </div>
	      
        <hr>
        <h3>Day 4 (April 29, 2025)</h3>
        <hr>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:00 - 09:05</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity"><b>Welcome</b></p></div>
          </div>

          <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">09:05 - 10:05</p></div>
          <div class="col-md-5 col-sm-5 col-xs-5 col-6"><p class="invited-speaker"><a href="https://eichlerlab.gs.washington.edu/evan.html">Evan E. Eichler</a></p></div>
          <div class="col-md-4 col-sm-4 col-xs-4 col-2">
		    <button type="button" class="btn-keynote" data-toggle="collapse" data-target="#keynote7"> Keynote Talk</button>
          </div>
          </div>

          <div class="row">
            <div class="col-md-3 col-sm3 col-xs-3 col-4"></div>
            <div class="col-md-9 col-sm9 col-xs-9 col-8"><div id="keynote7" class="collapse show">
        
              <p class="text">
                  <b>A pangenome perspective of structural variation</b>
                  <br><br>
                  <button class="btn-abstract" data-toggle="collapse" data-target="#collapsek7"><i class="fas fa-pencil-alt" aria-hidden="true"></i>Abstract</button>
                  <div class="collapse" id="collapsek7">
                    <p class="text">
          
The discovery and resolution of genetic variation is critical to understanding disease and evolution. I will present our most recent work to sequence and assemble telomere-to-telomere diverse human and nonhuman primate (NHP) genomes using both ultra-long and high-fidelity long-read sequencing technologies. The emerging data from hundreds of T2T human & ape genomes allows us, in principle, to reconstruct the evolutionary history of every base of our genome, revealing complex patterns of mutation and rapid diversification of genes and regions previously inaccessible by short-read sequencing. I will discuss algorithmic challenges to correctly representing such variation using graph-based approaches and potential solutions based on an understanding of the dynamic mutational processes shaping these regions at the species, population, and familial level. I will also present some recent tools that have been developed to leverage both graphs and pangenomes, as well as some approaches to integrate short-read sequencing data to improve future disease association in humans.
                    </p>
                  </div>
              </p>
              </div></div>
            </div>  

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:05 - 10:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Coffee Break</p></div>
          </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">10:20 - 12:20</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session13">Session 13: Genomic Variation and Privacy</button>
		<div id="session13" class="collapse show">
		<p class="text">
                <b>[PT] TX-Phase: Secure phasing of private genomes in a trusted execution environment</b>
                <br>
                <i>Natnatee Dokmai, Kaiyuan Zhu, Cenk Sahinalp and Hyunghoon Cho</i>
                <hr>

		<p class="text">
                <b>[PT] BayesRVAT: Bayesian aggregation of multiple annotations enhances rare variant association testing</b>
                <br>
                <i>Antonio Nappi, Na Cai and Francesco Paolo Casale</i>
                <hr>

		<p class="text">
                <b>[PT] ralphi: a deep reinforcement learning framework for haplotype assembly</b>
                <br>
                <i>Enzo Battistella, Anant Maheshwari, Barış Ekim, Bonnie Berger and Victoria Popic</i>
                <hr>

		<p class="text">
                <b>[PT] GEM-Finder: dissecting GWAS variants via long-range interacting cis-regulatory elements with differentiation-specific genes</b>
                <br>
                <i>Gyeongsik Park, Andrew J Lee, Sunwoo Min, Seyoung Jin and Inkyung Jung</i>
                <hr>

		<p class="text">
                <b>[PT] Dynamic mu-PBWT: Dynamic run-length compressed PBWT for biobank scale data</b>
                <br>
                <i>Pramesh Shakya, Ahsan Sanaullah, Degui Zhi and Shaojie Zhang</i>
                <hr>

		<p class="text">
                <b>[HT] Private information leakage from single-cell count matrices</b>
                <br>
                <i>Conor Walker, Xiaoting Li, Manav Chakravarthy, William Lounsbery-Scaife, Yoolim A. Choi, Ritambhara Singh and Gamze Gursoy</i>
                <hr>

		<p class="text">
                <b>[HT] The simplicity of protein sequence-function relationships</b>
                <br>
                <i>Yeonwoo Park, Brian Metzger and Joseph Thornton</i>
        
          	</div>		  
	  </div>
	  </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">12:20 - 13:30</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Lunch Break</p></div>
          </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">13:30 - 14:24</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session14">Session 14: Cancer Evolution</button>
		<div id="session14" class="collapse show">
		<p class="text">
                <b>[PT] Pharming: Joint clonal tree reconstruction of SNV and CNA evolution from single-cell DNA sequencing of tumors</b>
                <br>
                <i>Leah Weber, Anna Hart, Idoia Ochoa and Mohammed El-Kebir</i>
                <hr>

		<p class="text">
                <b>[PT] ScisTree2: An improved method for large-scale inference of cell lineage trees and genotype calling from noisy single cell data</b>
                <br>
                <i>Haotian Zhang, Yiming Zhang, Teng Gao and Yufeng Wu</i>
                <hr>

		<p class="text">
                <b>[PT] A partition function algorithm to evaluate inferred subclonal structures in single cell sequencing datasets</b>
                <br>
                <i>Farid Rashidi Mehrabadi, Erfan Sadeqi Azer, John Bridgers, Salem Malikic, A. Funda Ergun and S. Cenk Sahinalp</i>
        
          	</div>		  
	  </div>
          </div>

	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">14:25 - 15:19</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session15">Session 15: Lineage Reconstruction</button>
		<div id="session15" class="collapse show">
		<p class="text">
                <b>[PT] Tree reconstruction guarantees from CRISPR-Cas9 lineage tracing data using Neighbor-Joining</b>
                <br>
                <i>Sebastian Prillo, Kevin An, Wilson Wu, Ivan Kristanto, Matthew Jones, Yun Song and Nir Yosef</i>
                <hr>

		<p class="text">
                <b>[PT] Inferring cell differentiation maps from lineage tracing data</b>
                <br>
                <i>Palash Sashittal, Richard Zhang, Benjamin Law, Alexander Strzalkowski, Henri Schmidt, Adriano Bolondi, Michelle Chan and Ben Raphael</i>
                <hr>

		<p class="text">
                <b>[PT] Dynamic programming algorithms for fast and accurate cell lineage tree reconstruction from CRISPR-based lineage tracing data</b>
                <br>
                <i>Junyan Dai and Erin Molloy</i>
        
          	</div>		  
	  </div>
	  </div>

	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:20 - 15:40</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Coffee Break</p></div>
          </div>
	      
	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">15:40 - 16:34</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session16">Session 16: Spatial Omics</button>
		<div id="session16" class="collapse show">
		<p class="text">
                <b>[PT] TissueMosaic enables cross-sample differential analysis of spatial transcriptomics datasets through self-supervised representation learning</b>
                <br>
                <i>Sandeep Kambhampati, Luca D'Alessio, Fedor Grab, Stephen Fleming, Fei Chen and Mehrtash Babadi</i>
                <hr>

		<p class="text">
                <b>[PT] Joint imputation and deconvolution of gene expression across spatial transcriptomics platforms</b>
                <br>
                <i>Hongyu Zheng, Hirak Sarkar and Benjamin Raphael</i>
                <hr>

		<p class="text">
                <b>[PT] Unified integration of spatial transcriptomics across platforms</b>
                <br>
                <i>Ellie Haber, Ajinkya Deshpande, Jian Ma and Spencer Krieger</i>
        
          	</div>		  
	  </div>
	  </div>
	      
	  <div class="row">
	  <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">16:35 - 17:29</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8">
          	<button type="button" class="btn" data-toggle="collapse" aria-expanded="true" data-target="#session17">Session 17: Single-cell (II)</button>
		<div id="session" class="collapse show">
		<p class="text">
                <b><[PT] Synthetic control removes spurious discoveries from double dipping in single-cell and spatial transcriptomics data analyses/b>
                <br>
                <i>Dongyuan Song, Siqi Chen, Christy Lee, Kexin Li, Xinzhou Ge and Jingyi Jessica Li</i>
                <hr>

		<p class="text">
                <b>[PT] GeneCover: A combinatorial approach for label-free marker gene selection</b>
                <br>
                <i>An Wang, Stephanie Hicks, Donald Geman and Laurent Younes</i>
                <hr>

		<p class="text">
                <b>[PT] Integration and querying of multimodal single-cell data with product-of-experts VAE</b>
                <br>
                <i>Anastasia Litinetskaya, Maiia Shulman, Fabiola Curion, Artur Szalata, Alireza Omidi, Mohammad Lotfollahi and Fabian Theis</i>
        
          	</div>		  
	  </div>
	  </div>
	      
	  <div class="row">
          <div class="col-md-3 col-sm-3 col-xs-3 col-4"><p class="time">17:30 - 18:00</p></div>
          <div class="col-md-9 col-sm-9 col-xs-9 col-8"><p class="activity">Awards and Closing </p></div>
          </div>
	      
     *Time Zone: KST

      </div>
</div>

      </div>
	  
</div>


  </div> <!-- End container -->
</section> <!-- End section -->
<!-- End Program
    ==================================== -->
   <!-- 
<section class="team-skills section-sm" id="skills">
  <div class="container">
    <div class="col-12">
      <div class="team-skills-content center">

        <hr> <h2>Accepted Papers</h2> <hr>
      </div>
	  </div>

<ul>
<li> Ghanshyam Chandra and Chirag Jain. "Sequence to graph alignment using gap-sensitive co-linear chaining".</li>
<li>Nathan LaPierre, Boyang Fu, Steven Turnbull, Eleazar Eskin and Sriram Sankararaman. "Leveraging family data to design Mendelian Randomization that is provably robust to population stratification". </li>
<li>Yasamin Tabatabaee, Sebastien Roch and Tandy Warnow. "Statistically Consistent Rooting of Species Trees under the Multi-Species Coalescent Model". </li>
<li>Jiacheng Lin, Hanwen Xu, Addie Woicik, Jianzhu Ma and Sheng Wang. "Pisces: A combo-wise contrastive learning approach to synergistic drug combination prediction". </li>
<li>Uyen Mai, Eduardo Charvel and Siavash Mirarab. "MD-Cat: Phylogenetic dating under a flexible categorical model using Expectation-Maximization". </li>
<li>Louxin Zhang, Niloufar Abhari, Caroline Colijn and Yufeng Wu. "A Fast and Scalable Method for Inferring Phylogenetic Networks from Trees via Aligning the Ancestor Sequences of Taxa". </li>
<li>Xiaogen Zhou, Zhiqiang Li and Tong Tong. "DM-Net: A Dual-Model Network for Automated Biomedical Image Diagnosis". </li>
<li>Jim Shaw and Yun William Yu. "Seed-chain-extend alignment is accurate and runs in close to O(m log n) time for similar sequences: a rigorous average-case analysis". </li>
<li>Soufiane M.C. Mourragui, Marco Loog, Mirrelijn van Nee, Mark van de Wiel, Marcel J.T. Reinders and Lodewyk F.A. Wessels. "Percolate: an exponential family JIVE model to design DNA-based predictors of drug response". </li>
<li>Amir Joudaki, Alexandru Meterez, Harun Mustafa, Ragnar Groot Koerkamp, André Kahles and Gunnar Rätsch. "Aligning Distant Sequences to Graphs using Long Seed Sketches". </li>
<li>Daniel Baker and Ben Langmead. "Dashing 2: genomic sketching with multiplicities and locality-sensitive hashing". </li>
<li>Bing-Xue Du, Yi Xu, Siu-Ming Yiu, Hui Yu and Jian-Yu Shi. "MTGL-ADMET: A Novel Multi-Task Graph Learning Framework for ADMET Prediction Enhanced by Status-Theory and Maximum Flow". </li>
<li>Andrea Cracco and Alexandru I. Tomescu. "Extremely-fast construction and querying of compacted and colored de Bruijn graphs with GGCAT". </li>
<li>Sara C. Schulte, Alexander Dilthey and Gunnar W. Klau. "HOGVAX: Exploiting Peptide Overlaps to Maximize Population Coverage in Vaccine Design with Application to SARS-CoV-2". </li>
<li>Benjamin Maier and Kristoffer Sahlin. "Entropy predicts fuzzy-seed sensitivity". </li>
<li>Rıza Özçelik, Alperen Bağ, Berk Atıl, Melih Barsbey, Arzucan Ozgur and Elif Ozkirimli. "DebiasedDTA: Improving the Generalizability of Drug-Target Affinity Prediction Models". </li>
<li>Mahmudur Rahman Hera, N. Tessa Pierce-Ward and David Koslicki. "Deriving confidence intervals for mutation rates across a wide range of evolutionary distances using FracMinHash". </li>
<li>David Pellow, Lianrong Pu, Baris Ekim, Lior Kotlar, Ron Shamir and Yaron Orenstein. "Efficient minimizer orders for large values of k using minimum decycling sets". </li>
<li>Shuvom Sadhuka, Daniel Fridman, Bonnie Berger and Hyunghoon Cho. "Accurate evaluation of transcriptomic re-identification risks using discriminative sequence models". </li>
<li>Jason Fan, Jamshed Khan, Giulio Pibiri and Rob Patro. "Spectrum preserving tilings enable sparse and modular reference indexing". </li>
<li>Omar Ahmed, Massimiliano Rossi, Christina Boucher and Ben Langmead. "Efficient taxa identification using a pangenome index". </li>
<li>Baris Ekim, Kristoffer Sahlin, Paul Medvedev, Bonnie Berger and Rayan Chikhi. "hifimap: Efficient low-divergence mapping of long reads in minimizer space". </li>
<li>Stefan Ivanovic and Mohammed El-Kebir. "Modeling and Predicting Cancer Clonal Evolution with Reinforcement Learning". </li>
<li>Palash Sashittal, Henri Schmidt, Michelle Chan and Benjamin Raphael. "Startle: a star homoplasy approach for CRISPR-Cas9 lineage tracing". </li>
<li>Yudi Zhang, Ha Vu, Karin Dorman and Geetu Tuteja. "Unsupervised Deep Peak Caller for ATAC-seq". </li>
<li>Ahsan Sanaullah, Degui Zhi and Shaojie Zhang. "Minimal Positional Substring Cover: A Haplotype Threading Alternative to Li & Stephens Model". </li>
<li>Yunheng Han and Erin Molloy. "TREE-QMC: Improving quartet graph construction for scalable and accurate species tree estimation from gene trees". </li>
<li>Aditya Gorla, Sriram Sankararaman, Esteban Burchard, Jonathan Flint, Noah Zaitlen and Elior Rahmani. "Phenotypic subtyping via contrastive learning". </li>
<li>Shikha Mallick and Sahely Bhadra. "CDGCN: Conditional de novo Drug generative model using Graph Convolution Networks". </li>
<li>Hao Chen, Dongshunyi Li and Ziv Bar-Joseph. "Cell segmentation for high-resolution spatial transcriptomics".</li>
<li>Regev Schweiger and Richard Durbin. "Ultra-fast genome-wide inference of pairwise coalescence times".</li>
<li>Weizhong Zheng, John H.C. Fong, Yuk Kei Wan, Athena H.Y. Chu, Yuanhua Huang, Alan S.L. Wong and Joshua W.K. Ho. "Translation rate prediction and regulatory motif discovery with multi-task learning".</li>
<li>Runpeng Luo and Yu Lin. "VStrains: De Novo Reconstruction of Viral Strains Via Iterative Path Extraction From Assembly Graphs".</li>
<li>Ziqi Chen, Martin Min, Hongyu Guo, Chao Cheng, Trevor Clancy and Xia Ning. "T-Cell Receptor Optimization with Reinforcement Learning and Mutation Polices for Precision Immunotherapy".</li>
<li>Rajagopal Venkatesaramani, Zhiyu Wan, Bradley Malin and Yevgeniy Vorobeychik. "Enabling Trade-offs in Privacy and Utility in Genomic Data Beacons and Summary Statistics".</li>
<li>Ardalan Naseri, William Yue, Shaojie Zhang and Degui Zhi. "FastRecomb: Fast inference of genetic recombination rates in biobank scale data".</li>
<li>Spencer Krieger and John Kececioglu. "Computing shortest hyperpaths for pathway inference in cellular reaction networks".</li>
<li>Claire McWhite and Mona Singh. "Vector-clustering Multiple Sequence Alignment: Aligning into twilight zone of protein sequence similarity with protein language models".</li>
<li>Rohit Singh, Alexander Wu, Anish Mudide and Bonnie Berger. "Unraveling causal gene regulation from the RNA velocity graph using Velorama".</li>
<li>Xinhao Liu, Ron Zeira and Benjamin J. Raphael. "PASTE2: Partial Alignment of Multi-slice Spatially Resolved Transcriptomics Data".</li>
<li>Yuelin Liu, Xuan C. Li, Farid Rashidi Mehrabadi, Alejandro A. Schäffer, Drew Pratt, David R. Crawford, Salem Malikić, Erin K. Molloy, Vishaka Gopalan, Stephen M. Mount, Eytan Ruppin, Kenneth Aldape and S. Cenk Sahinalp. "Single-cell methylation sequencing data reveals succinct metastatic migration histories and tumor progression models".</li>
<li>Vtalii Stebliankin, Azam Shirali, Prabin Baral, Prem Chapagain and Giri Narasimhan. "PIsToN: Evaluating Protein Binding Interfaces with Transformer Networks".</li>
<li>Chihao Zhang, Yiling Elaine Chen, Shihua Zhang and Jingyi Jessica Li. "Information-theoretic Classification Accuracy: A Criterion that Guides Data-driven Combination of Ambiguous Outcome Labels in Multi-class Classification".</li>
<li>Zhihang Hu, Qinze Yu, Yucheng Guo, Taifeng Wang, Irwin King, Xin Gao, Le Song and Yu Li. "Drug Synergistic Combinations Predictions via Large-Scale Pre-Training and Graph Structure Learning".</li>

</ul> <br />

    </div>    
  </div> <!-- End container -->
</section> <!-- End section -->

<!-- begin Footer Area
    ========================================== -->
<footer id="footer" class="bg-one">
  <div class="top-footer">
    <p>Any questions? Email organizers at <b>recomb2025@gmail.com</b></p>
  </div>
  <div class="footer-bottom">
    <h5>Copyright 2025. All rights reserved.</h5>
    <h6>Design and Developed by Themefisher</h6>
  </div>
</footer> <!-- end footer -->
<!-- end Footer Area
    ========================================== -->

<!-- Essential Scripts
    =====================================-->
    <!-- Main jQuery -->
    <script src="plugins/jquery/jquery.min.js"></script>
    <!-- Google Map -->
    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBu5nZKbeK-WHQ70oqOWo-_4VmwOwKP9YQ"></script>
    <script  src="plugins/google-map/gmap.js"></script>

    <!-- Form Validation -->
    <script src="plugins/form-validation/jquery.form.js"></script> 
    <script src="plugins/form-validation/jquery.validate.min.js"></script>
    
    <!-- Bootstrap4 -->
    <script src="plugins/bootstrap/js/bootstrap.min.js"></script>
    <!-- Parallax -->
    <script src="plugins/parallax/jquery.parallax-1.1.3.js"></script>
    <!-- lightbox -->
    <script src="plugins/lightbox2/dist/js/lightbox.min.js"></script>
    <!-- Owl Carousel -->
    <script src="plugins/slick/slick.min.js"></script>
    <!-- filter -->
    <script src="plugins/filterizr/jquery.filterizr.min.js"></script>
    <!-- Smooth Scroll js -->
    <script src="plugins/smooth-scroll/smooth-scroll.min.js"></script>
    
    <!-- Custom js -->
    <script src="js/script.js"></script>

  </body>
  </html>
